### DISCUSSION TITLE
Ajouter pré-requis version Mysql/MariaDb dans la page INSTALL.txt

### DISCUSSION THREAD

**Discussion by acdct** [JeromeDevome/GRR#247] [Idées]
Bonjour,

Est-il possible d'ajouter dans la page INSTALL.txt, au niveau des pré-requis, la version MySQL-MariaDb à posséder ?

J'en fais la demande car chez nous la version de la BDD installée ne permet pas de passer en 3.5.0 et supérieure.

Je pensais pouvoir au moins installer la 3.5.0 et 3.5.1 car il n'était spécifié que la version de PHP à posséder, à savoir la 5.6.0.
Le changement d'interclassement utf8mb4_unicode_ci bloque la mise à jour car n'existe pas dans ma version BDD, du coup je reste bloqué sur la 3.4.3b en attendant une mise à jour du serveur.

Merci beaucoup

**Comment by ynaessens** [comment]
Bonjour,
merci pour votre remarque.
C'est précisé par le commit https://github.com/JeromeDevome/GRR/commit/ae0898641ed790adbd7cc92318f3f8d8128c62c7
S'il y a encore des serveurs en anciennes version de MySQL, il faudrait penser à rétroporter certaines modifications ?
Cordialement,
YN

**Reply by acdct** [reply]
Merci beaucoup pour cet ajout.
Je reste pour le moment sous la version 3.4.3b en attendant l'aval pour une mise à jour serveur pour pouvoir mettre la dernière version de GRR.
Bonne journée.
